wiki/Tournaments/otst!/2019/en.md

```diff
@@ -0,0 +1,224 @@
+| **we are bad** | **![][flag_US] [Stressed](/users/8112504)**, ![][flag_GB] [- Akumi -](/users/10528604), ![][flag_US] [onetabby](/users/9508109), ![][flag_US] [Cuckweezy](/users/7154358), ![][flag_US] [Atem](/users/7143394), ![][flag_US] [Totoxri](/users/7221207) |
+| **White Dawgs** | **![][flag_AU] [-Machine-](/users/5459981)**, ![][flag_AU] [Inertialboss](/users/2245285), ![][flag_AU] [Burillion](/users/2055677), ![][flag_GB] [Eriza](/users/11251641), ![][flag_AU] [freddofrog](/users/6292111), ![][flag_AU] [ImActuallyCaleb](/users/5167008) |
+
+-----------------------
+
+## Mappools
+
+*First mappools (Group Stage) will be revealed on 2019-02-16 at 14:00 UTC.*
+
+------------------------------------------------------------------------
+
+## Ruleset
+
+### General Rules
+
+1. There will be 6 mappools in total - Group stage, Round of 32, Round of 16, Quarterfinals, Semifinals, and Finals. 
+2. Mappool sizes will progressively get bigger each round after Ro16. Starting with a 4/2/2/2/2/3 format. This format will be explained as follows:
+   - 4 NoMod
+   - 2 Hidden
+   - 3 HardRock
+   - 2 DoubleTime
+   - 2 FreeMod
+   - 3 Tiebreakers
+3. Group stage will be round-robin, followed by a double elimination bracket.
+4. Maps will be no longer than 2 minutes drain time.
+5. The winner of `!roll` will ban and pick first.
+6. Teams get **one** ban in group stage, Round of 32 and Round of 16. The rest of final stage, teams will get **two** bans.
+7. Tiebreakers will be a BO3 instead of a single match.
+   - `!roll`, higher roll picks first during a tiebreaker.
+
+### Match Format
+
+1. In Group Stage, teams need to win 4 beatmaps to win a match. (Best-of-7)
+2. In the Round of 32 and the Round of 16, teams need to win 5 beatmaps to win a match. (Best-of-9)
+3. In Quarterfinals and the Semifinals, teams need to win 6 beatmaps to win a match. (Best-of-11)
+4. In the **Finals and Grand Finals**, teams need to win 7 beatmaps to win a match. (Best-of-13)
+   - There will be a bracket reset match if teams from losers bracket wins the first match.
+
+### Registration
+
+1. If a team member needs to be replaced after signing up, please contact a host **before** registration ends. No rosters can be modified after registration ends.
+2. Each member of participacing team must be in the Discord server to verify their participation. 
+3. If a player changes their name, they (or a team captain) needs to inform a tournament host or admin as soon as possible to avoid misunderstandings in a match.
+4. If a player exhibits suspicious behavior, including but not limited to suspected multi-accounting or cheating, their participation in the tournament might be prohibited, subject to tournament host discretion.
+5. Accounts that were recently unbanned or suspended within 12 months prior to February 9th will not be allowed to participate in otst!.
+
+### Scheduling
+
+- **All times in the match schedules are in UTC**
```

Comment:
```suggestion
- **All times in the match schedules are in UTC.**
```

Reply:
seems like this shouldn't be a list? applies to some of the other notes below headers